tagging/apply_tags.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+import argparse
+import logging
+from plumbum.cmd import docker
+from docker_runner import DockerRunner
+from images_hierarchy import ALL_IMAGES
+
+
+logger = logging.getLogger(__name__)
+
+
+def get_all_taggers(short_image_name):
+    taggers = []
+    while short_image_name is not None:
+        image_description = ALL_IMAGES[short_image_name]
+        taggers = image_description.taggers + taggers
+        short_image_name = image_description.parent_image
+    return taggers
+
+
+def apply_tags(short_image_name, owner):
+    logger.info(f"Applying tags for image: {short_image_name}")
+    taggers = get_all_taggers(short_image_name)
+
+    image = f"{owner}/{short_image_name}:latest"
+
+    with DockerRunner(image) as container:
```

Comment:
@romainx this allowed me to reuse the image created so right now I'm only creating container once and then reusing it to calculate tag values.